enemy-test/src/enemy_monster.c

```diff
@@ -24,71 +24,58 @@ void display_enemy(ENEMY* unit, UINT8 xpos, UINT8 ypos) {
 	
 	// Initialize left sprite
 	set_sprite_tile(unit->sprite_id, unit->enemy_sprite_l);
-	move_sprite(unit->sprite_id, xpos, ypos);
+	move_sprite(unit->sprite_id, xpos - 8, ypos - 16);
 	
 	// Initialize right sprite
 	set_sprite_tile(unit->sprite_id + 1, unit->enemy_sprite_r);
-	move_sprite(unit->sprite_id + 1, xpos + 8, ypos);
+	move_sprite(unit->sprite_id + 1, xpos, ypos - 16);
 }
 
 // Play death sequence (blinking), then make the enemy disappear
 void enemy_death(ENEMY* unit) {
-	// Blinking animation
+	// Blinking animation - disabled
+	/*
 	UINT8 blinking_cycles = 0;
 	UINT8 cycle_state = 1;
 	
 	UINT8 former_ypos = unit->ypos; // Retain former y pos for the blinking animation
 	
-	while (blinking_cycles < 10)
-	{
-		wait_vbl_done();
-		wait_vbl_done();
-		wait_vbl_done();
-		wait_vbl_done();
-		wait_vbl_done();
-		wait_vbl_done();
-		
-		if (cycle_state) // Disappears
-		{
+	while (blinking_cycles < 10) {	
+		if (cycle_state) { // Disappears
 			move_sprite(unit->sprite_id, unit->xpos, 0);
 			move_sprite(unit->sprite_id+1, unit->xpos+8, 0);
 			cycle_state = 0;
-		}
-		else
-		{
+		} else {
 			move_sprite(unit->sprite_id, unit->xpos, former_ypos);
 			move_sprite(unit->sprite_id+1, unit->xpos+8, former_ypos);
 			cycle_state = 1;
 			blinking_cycles++;
 		}
-	}
+	}*/

```

Comment:
This definitely isn't what I would call fixing something.
You need to add something akin to a countdown.